cmd/geth/flag.go

```diff
@@ -842,31 +846,37 @@ func mustMakeSufficientConfiguration(ctx *cli.Context) *core.SufficientChainConf
 	// Initialise chain configuration before handling migrations or setting up node.
 	config.ID = getChainConfigIDFromContext(ctx)
 	config.Name = getChainConfigNameFromContext(ctx)
-	config.ChainConfig = MustMakeChainConfig(ctx).SortForks()
+	config.ChainConfig = MustMakeChainConfigFromDefaults(ctx).SortForks()
 	config.ParsedBootstrap = MakeBootstrapNodesFromContext(ctx)
 
+	if isTestMode(ctx) {
```

Comment:
In light of #254 I am going to leave it because we will be changing the way gets treats external configs and this will no longer be an issue; I prefer to leave it to maintain balance with external configuration settings.